rfps/uncollateralized-stablecoin.md

```diff
@@ -0,0 +1,56 @@
+
+Stablecoins are cryptocurrencies where the price is designed to be pegged (=fixed exchange rate) to a cryptocurrency, fiat money, or to exchange-traded commodities. Seigniorage-style, uncollateralized or algo stablecoin is a token that uses algorithms to control the circulating supply to get a stable value of the asset. In general the price volatility of cryptocurrencies is one of the biggest barriers to widespread adoption. Stablecoins therefore have become a key component of DeFi applications. However, all successful existing stable coins (e.g. DAI, USDT, USDC, etc) are asset backed. Therefore they are subject to the same volatility and risk associated with the backing asset and the underlying process. Some of the potentially issues based on this are:
+- Additional trust assumptions (e.g. USDT)  
+- Scalability issues (restricted by the underlying asset) 
+- Devalue under massive crashes in the underlying assets 
+
+Purely algorithmic stablecoin would overcome these risks. Additionally it would be fairly simple to peg an algorithmic stablecoin to different assets (USD, EUR) or in the future even to a consumer price index (CPI). However, until now there is no purely algorithmic stablecoin that didn’t introduce signification additional disadvantages or was able to maintain it’s peg for a long period of time (e.g. AMPL, ESD, DSD, BAC, NuBits). 
+
+The goal of this RFP is to research and create new uncollateralized stablecoin mechanisms and implement these as ink! Smart contract or pallets. The biggest research question hereby is how to efficiently decrease the supply of the token.   
+
+**Useful resources:** 
+- https://www.basis.io/basis_whitepaper_en.pdf  
+- https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2425270 
+- https://www.youtube.com/watch?v=bnC5RQsaAXQ 
+- https://econteric.com/stablecoin/algo/ 
+
+## Deliverables :nut_and_bolt:
+
+The milestones below are just an initial draft. The milestones can be structured completely differently and the implementation can also leverage other tools and infrastructure as long as the overall goal of the RFP is met.
+
+* **Total Estimated Duration:** 2 month
+* **Full-time equivalent (FTE):**  1
+* **Total Costs:** 20k
+
+### Milestone 1 - Research & Design 
+
+* **Estimated Duration:** Duration of milestone 1 
+* **FTE:**  Amount of time (in days) required for a single person to complete this milestone
+* **Costs:** Amount of Payment in USD for milestone 1
+
+
+| Number | Deliverable | Specification | 
+| ------------- | ------------- | ------------- |
+| 1. | Overview of existing solutions | Create an overview of existing and previous uncollateralized stablecoin solutions and why they failed. |  
+| 2. | Increasing Supply | Develop a mechanism to deal with an increasing demand for the cryptocurrency and analyses potential attack vectors | 
+| 3. | Decreasing Supply | Develop a mechanism to deal with an decreasing demand for the cryptocurrency and analyses potential attack vectors| 
```

Comment:
I feel like whether it's worth it to go ahead and do milestone 2 at all may depend too much on this being done right, and like the likelihood of a team delivering something that looks reasonable at first glance but is inherently broken for this part is too high to make the implementation part of the same grant.

This may be a good opportunity to have a research-only rfp/grant.

Reply:
Makes sense. I will update it